server/ormconfig.ts

```diff
@@ -0,0 +1,54 @@
+import { ConnectionOptions } from 'typeorm';
+import * as path from 'path';
+import * as dotenv from 'dotenv';
+import * as fs from 'fs';
+
+function buildConnectionOptions(
+  filePath: string,
+  env: string | undefined,
+): ConnectionOptions {
+  const data: any = dotenv.parse(fs.readFileSync(filePath));
+  const databaseName: string = env === 'test' ? 'tooljet_test' : data.PG_DB;
+
+  return {
+    type: 'postgres',
+    database: databaseName,
+    port: data.PG_PORT || 5432,
+    username: data.PG_USER,
+    password: data.PG_PASS,
+    host: data.PG_HOST,
+    entities: ['dist/**/*.entity{.ts,.js}'],
+    synchronize: false,
+    migrationsRun: false,
+    logging: data.ORM_LOGGING || false,
```

Comment:
`data.ORM_LOGGING` will be a string, right?

Reply:
Yeah, so logger options [accepts](https://github.com/typeorm/typeorm/blob/master/src/logger/LoggerOptions.ts#L4) either a string or boolean. We'd either specify in those options else default as false. Wdyt?

Reply:
Sounds good 👍